mono/metadata/marshal.h

```diff
@@ -761,13 +761,16 @@ MonoBoolean
 ves_icall_System_Runtime_InteropServices_Marshal_IsComObject (MonoObjectHandle object, MonoError *error);
 
 gint32
-ves_icall_System_Runtime_InteropServices_Marshal_ReleaseComObjectInternal (MonoObject* object);
+ves_icall_System_Runtime_InteropServices_Marshal_ReleaseComObjectInternal (MonoObjectHandle object, MonoError *error);
 
 MonoObjectHandle
 ves_icall_System_ComObject_CreateRCW (MonoReflectionTypeHandle ref_type, MonoError *error);
 
 void
-ves_icall_System_ComObject_ReleaseInterfaces(MonoComObject* obj);
+mono_System_ComObject_ReleaseInterfaces (MonoComObjectHandle obj);
```

Comment:
I'm curious why this one is global while `mono_IUnknown_Release` is static, and an accessor for `QueryInterface` shouldn't exist at all.

Reply:
Could be maybe more consistent. I can look again.

Reply:
I added all 3 helpers now. I think one is only used once, or zero. But there was room for consistency, agreed.